contents/blog/how-to-fix-your-growth-team.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Why your growth team isn't winning (and how to fix it)
```

Comment:
Did a second pass, seems to broadly make sense!

I think my biggest remaining issue is the negative slant "why your growth team isn't winning (and how to fix it)".

It's like if a personal trainer wrote a blog post saying "why you're fat."  ...sure, but now you're not the person I want to hear from now.

What's a more positive title we might take here?
- Helping your growth team hit the high notes?
- Growing your Growth team's impact?

etc

Reply:
Hmm interesting. I like this title as it sets up the rest of the post nicely i.e. the problems you need to dive into.

I'll give it some thought though and see if I can come up with something less negative

> It's like if a personal trainer wrote a blog post saying "why you're fat." ...sure, but now you're not the person I want to hear from now.

I think it's closer to "why you're not losing weight", which is not as negative :)

Reply:
How about something along the lines of: 
- The formula of a successful growth team

One concern would be that a title like that may mean changing the "problem - how to fix it" format of the blog  (and I quite like it!).

I'll update this PR with an update and see how it works